src/herder/HerderSCPDriver.h

```diff
@@ -138,6 +138,10 @@ class HerderSCPDriver : public SCPDriver
     // returns false on error
     bool toStellarValue(Value const& v, StellarValue& sv);
 
+    // validate close time as much as possible
+    bool checkCloseTime(uint64_t slotIndex, uint64_t lastCloseTime,
```

Comment:
Commit order is a bit strange here: the renaming of this function happens in the next commit, although the updated name is used here. Also, this change is reverted in the next commit anyway, and the function remains private.

Reply:
no: there is `checkCloseTime` in `HerderSCPDriver` (that I made public), the one that got renamed is in `HerderImpl`